plugins/elb/elbLoggingEnabled.js

```diff
@@ -0,0 +1,61 @@
+var async = require('async');
+var helpers = require('../../helpers');
+var describeLoadBalancerAttributes = require('../../collectors/elb/describeLoadBalancerAttributes.js');
```

Comment:
This isn't needed here. `describeLoadBalancerAttributes` should be part of the `collect` object created earlier.